src/FluentAvalonia/UI/Controls/MenuFlyout/MenuFlyoutInteractionHandler.cs

```diff
@@ -70,9 +70,10 @@ public virtual void Attach(FAMenuFlyoutPresenter menu)
             window.Deactivated += WindowDeactivated;
         }
 
-        if (_root is TopLevel tl && tl.PlatformImpl != null)
-            tl.PlatformImpl.LostFocus += TopLevelLostPlatformFocus;
-
+        // v2p6.1: Don't have access to this anymore...
+        //if (_root is TopLevel tl && tl.PlatformImpl != null)
+        //    tl.PlatformImpl.LostFocus += TopLevelLostPlatformFocus;
```

Comment:
I wonder if we should push to add Deactivated from WPF:

https://learn.microsoft.com/en-us/dotnet/api/system.windows.window.deactivated?view=windowsdesktop-7.0

This seems like necessary information to have.

Reply:
`Deactivated` is available on `WindowBase`.

Actually, thinking about this, I don't think this is actually important. This code was copied from upstream and then modified for `FAMenuFlyout`, but FAMenuFlyout is always light-dismiss so any deactivation of the top level should end up closing the flyout anyway. This file needs a refactor anyway, I just haven't felt like doing it yet.